src/components/manage/index.js

```diff
@@ -444,8 +444,15 @@ export class Manage extends Component {
     }
     const { start_time, end_time } = await getCrowdsaleStartAndEndTimes(...params).call()
     let crowdsaleInfo = await getCrowdsaleInfo(...params).call()
-    if (crowdsaleInfo && crowdsaleInfo.hasOwnProperty('is_finalized')) {
-      crowdsaleInfo.is_finalized = crowdsaleInfo[4]
+
+    if (isMintedCappedCrowdsale) {
+      if (crowdsaleInfo && !crowdsaleInfo.hasOwnProperty('is_finalized')) {
```

Comment:
@mariano-aguero maybe, we could move `if (crowdsaleInfo && !crowdsaleInfo.hasOwnProperty('is_finalized')) {` to outside:
```
if (crowdsaleInfo && !crowdsaleInfo.hasOwnProperty('is_finalized')) {
  if (isMintedCappedCrowdsale) {
    crowdsaleInfo.is_finalized = crowdsaleInfo[3]
  } else if (isDutchAuction) {
    crowdsaleInfo.is_finalized = crowdsaleInfo[4]
  }
}
```
?